src/AppShed/Extensions/SpreadsheetBundle/Controller/ReadController.php

```diff
@@ -160,6 +184,36 @@ public function documentAction(Request $request)
         }
     }
 
+    function readGeocode($address){
```

Comment:
Can`t understand how works your service as Reference... So, temporary i created very simple service that drops to ReadController as argument. P.S Why we use service config in XML?